include/lsst/afw/cameraGeom/Detector.h

```diff
@@ -210,17 +210,17 @@ class Detector {
      *
      * @note the CameraSysPrefix version needs the detector name, which is why this is not static.
      */
-    CameraPoint makeCameraPoint(geom::Point2D point,  ///< 2-d point
-                                CameraSys cameraSys   ///< coordinate system
+    CameraPoint makeCameraPoint(geom::Point2D const &point,  ///< 2-d point
+                                CameraSys const &cameraSys   ///< coordinate system
                                 ) const {
         return CameraPoint(point, cameraSys);
     }
 
     /**
      * Make a CameraPoint from a point and a camera system prefix
      */
-    CameraPoint makeCameraPoint(geom::Point2D point,             ///< 2-d point
-                                CameraSysPrefix cameraSysPrefix  ///< coordinate system prefix
+    CameraPoint makeCameraPoint(geom::Point2D const &point,             ///< 2-d point
+                                CameraSysPrefix const &cameraSysPrefix  ///< coordinate system prefix
```

Comment:
Our coding standards don't specify a right way to do this, so feel free to leave this as it is, but my personal preference is for the syntax to be `geom::Point2D const & point` with a space after &. This is more in line with the rest of our code base, and is visually distinctive from address of a variable.